### PR TITLE
Padim exclude gauss

### PR DESCRIPTION
**Pull request by wlinds**
Set gaussian_blur as arg (bool) for padim.predict method (defaults to True)

### CODE REVIEW

File: anodet/padim.py [hunk 1]
```diff
@@ -137,9 +137,13 @@ def predict(self, batch: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
 
         score_map = F.interpolate(patch_scores.unsqueeze(1), size=batch.shape[2],
                                   mode='bilinear', align_corners=False).squeeze()
+
```
Comment (antonemanuel) [inline anodet/padim.py:140]: Why are there two lines here?

File: anodet/feature_extraction.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 import torch
 import torch.nn.functional as F
-from torchvision import models
+from torchvision.models import resnet18, ResNet18_Weights, wide_resnet50_2, Wide_ResNet50_2_Weights
```
Comment (antonemanuel) [inline anodet/feature_extraction.py:7]: Did you mean to include this in the pr? I saw that you had another commit with these changes. If you bother I think it's nicer with two different pr's, to keep them simple.

**Review by antonemanuel — CHANGES_REQUESTED** [review]
See comments in code

File: anodet/padim.py (hunk 1 above)
Comment (wlinds) [reply anodet/padim.py:140]: linter

**Pull request merged**